lib/SelectedObjectsProvider/index.js

```diff
@@ -6,7 +6,7 @@ const SelectedObjectsContext = React.createContext({});
 
 const SelectedObjectsProvider = ({ children }) => {
```

Comment:
Something that is starting to bug me with this name is why did we add "objects" to the name? 😂 

Why not something like `SelectionProvider`?